core/Url.php

```diff
@@ -668,13 +668,17 @@ protected static function getHostsFromConfig($domain, $key)
      */
     public static function getHostFromUrl($url)
     {
-        $parsedUrl = parse_url($url);
+        if (empty($url)) {
```

Comment:
should we just check for `is_string` here maybe? Not sure if that makes sense? I suppose host or Url can't be 0?

Reply:
makes sense